README.md

```diff
@@ -69,15 +69,42 @@
 
 ### Version
 
-```shell
+```console
 > mycoffee --version
 
 1.4
 ```
 
+### Info
+
+```console
+> mycoffee --info
+
+ __  __          ____          __   __
+|  \/  | _   _  / ___|  ___   / _| / _|  ___   ___
+| |\/| || | | || |     / _ \ | |_ | |_  / _ \ / _ \
+| |  | || |_| || |___ | (_) ||  _||  _||  __/|  __/
+|_|  |_| \__, | \____| \___/ |_|  |_|   \___| \___|
+         |___/
+
+__     __    _     _  _
+\ \   / / _ / |   | || |
+ \ \ / / (_)| |   | || |_
+  \ V /   _ | | _ |__   _|
+   \_/   (_)|_|(_)   |_|
+
+
+
+MyCoffee is a command-line tool for coffee enthusiasts who love brewing with precision.
+It helps you calculate the perfect coffee-to-water ratio for various brewing methods,
```

Comment:
We can elaborate this more: `perfect coffee-to-water ratio, temperature, ...,`

Reply:
You are right, but since we need to update other parts (like `README.md`, `setup.py`, etc.), I believe we can handle that in another PR.